lib/Models/Terria.ts

```diff
@@ -1241,6 +1241,47 @@ export default class Terria {
     );
   }
 
+  // This method is created to support unit tests.
+  async loadMapItems(model: BaseModel, errors: TerriaError[]) {
+    (await (<any>model).loadMapItems()).pushErrorTo(errors);
+  }
```

Comment:
Not sure why this is needed, also you shouldn't type cast to `any` here - you can use `MappableMixin.isMixedInto`.

Instead of using `spyOn(terria, 'loadMapItems')` - you can check if a model's map items as been loaded by checking `loadMapItemsResult` is defined

Reply:
This function is needed because I've revised the unit test so that it spies on `loadMapItems` but does not actually call through -- Calling through will make unit test unnecessarily complicated.